hack/run-ci-e2e-test.sh

```diff
@@ -87,12 +87,14 @@ fi
 
 # OPERATOR_IMAGE defines where the WMCO image to test with is located. If $OPERATOR_IMAGE is already set, use its value.
 # Setting $OPERATOR_IMAGE is required for local testing.
-# In OpenShift CI $IMAGE_FORMAT will be set to a value such as registry.svc.ci.openshift.org/ci-op-<input-hash>/stable:${component}
-# We need to replace ${component} with the name of the WMCO image.
-# The stable namespace is used as it will use the last promoted version when the test is run in an repository other than this one.
-# When running in this repository the image built in the pipeline will be used instead.
-# https://steps.svc.ci.openshift.org/help/ci-operator#release
-OPERATOR_IMAGE=${OPERATOR_IMAGE:-${IMAGE_FORMAT//\/stable:\$\{component\}//stable:windows-machine-config-operator-test}}
+# In CI $OPERATOR_IMAGE environment variable will be declared through a dependency in test references declared at
+# https://github.com/openshift/release/tree/master/ci-operator/step-registry/windows/e2e/operator/test
+if [[ "$OPENSHIFT_CI" = "true" ]]; then
+  if [[ -z "$OPERATOR_IMAGE" ]]; then
```

Comment:
I suggest checking for this even for local testing.

Reply:
updated PTAL. The tests passed the installation stage in previous run, verified through the test pod logs.